bin/tic_tac_toe.rb

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+
+require_relative '../lib/game'
+require_relative '../lib/game_setting/game_setting_setter'
+require_relative '../lib/game_setting/setting_types'
+
+include SettingTypes
+
+game_settings = GameSettingSetter.new()
+game_settings.create_settings
```

Comment:
Something we could consider here is to return `settings` from `create_settings` and retrieve the specific settings from this returned value. It makes the code much easier to read and reason about when return values from methods instead of calling functions that have side effects and reaching into the instance variable to get the settings.